### DISCUSSION TITLE
Find duplicate entries in a column in PostgreSQL graphql hasura?

### DISCUSSION THREAD

**Discussion by agam-singsys** [hasura/graphql-engine#7003] [General]
How do i find the Duplicate Values in One Column in PostgreSQL GraphQL Hasura. i know there is a Groupby in SQL but do i do in this case?

**Comment by esseswann** [comment]
There is no `group by` equivalent in Hasura. You can create a [view](https://hasura.io/docs/latest/graphql/core/guides/postgres/views.html) and expose it through Hasura
